community/sustainability_effort.md

```diff
@@ -0,0 +1,69 @@
+
+### Finding code to work on
+
+If you have a business need, definitely feel free to work on that. Otherwise, you can find an Issue and assign it to yourself. If you're a new developer, one option is to pick one up that has the 
+label `good-first-issue`. Otherwise, you can pick up an Issue labeled `P1` or `P2`. 
+
+### Finding a PR to review
+
+We will first prioritize review of any PRs which have the "prioritized" label, meaning they were authored by a participant of this sytem (we can verify that they in fact are by github ID). We need to make 
+sure each of these has an Assignee. An Assignee is a Reviewer/Approver/Lead who owns reviewing the PR. PRs with this label should be given an initial review within a week and should respond to each question 
+or modification from the author within a week.
+
+Next, look for the Pull Requests which have no existing Assignee. 
+
+In the case of a Reviewer being the Assignee, once the Reviewer has approved the PR, they can request a review from one or more Approvers.
+
+### Authoring PRs
+
+For participants of the system, any PRs that you author you can apply a "prioritized" label to.
+
+### Bug Triaging
+
+We need to make sure that all new bugs are seen by somebody so that we can identify the highest priority ones. There are labels `P0`, `P1`, `P2`, and `P3` that should be applied, in which `P0` is considered
+highest priority and needs immediate attention, followed by `P1`, `P2`, and then `P3`. The "bug" label can be removed if the issue is determined to be user error. The label `more-information-needed` can be added 
+if more information is needed from the author to in order to determine whether it's a bug or its priority. If there's a new `P0` bug, notify the [#argo-wf-contributors](https://cloud-native.slack.com/archives/C0510EUH90V) slack channel.
+
+Any bugs that have >= 5 "thumbs up" reactions should be labeled `P1`. Any bugs with 3-4 "thumbs up" should be labeled `P2`. (Bugs can be sorted by "thumbs up").
+
+## Participants
+
+(if you'd like to join, just add your name here and submit in a PR)
+
+| Name                      | GitHub ID                                               |
+|---------------------------|---------------------------------------------------------|
+| Julie Vogelman            | [juliev0](https://github.com/juliev0)                   |
+| Saravanan Balasubramanian | [sarabala1979](https://github.com/sarabala1979)         |
+| Yuan Tang                 | [terrytangyuan](https://github.com/terrytangyuan)       |
+| Alan Clucas               | [Joibel](https://github.com/Joibel)                     |
+| Isitha Subasinghe         | [isubasinghe](https://github.com/isubasinghe)           |
+| Jason Meridth             | [jmeridth](https://github.com/jmeridth)                 |
+| Shuangkun Tian            | [shuangkun](https://github.com/shuangkun)               |
```

Comment:
> Per the current system as is listed here, Shuangkun would not be eligible as Argoproj Membership is a prerequisite

Can you give me some advices to be a member. Thanks! I am an owner of a product in AlibabaCloud named Managed Argo Workflow(like aws MWAA)， which helps user have a better use in mlops、large scale、cicd scene and we also have many requests from users in product. So I want to more participate in community work to help the product work well and also let my PRs quickly reviewed.

Reply:
good catch @agilgur5. @shuangkun can you create an Issue in the argoproj repo?: https://github.com/argoproj/argoproj/issues similar to others you'll find in there that read "REQUEST: New membership for x"?

Reply:
It looks like you've authored 10 PRs - I don't think there should be an issue with getting it merged, although need to find out if maybe we could do it prior to the usual membership meeting which only occurs once every 3 months.

Reply:
Thanks!  It has been merged 8. I'll try to get them all merged and continuous to submit prs.

Reply:
> Thanks! It has been merged 8. I'll try to get them all merged and continuous to submit prs.

https://github.com/argoproj/argoproj/issues/266    I have create a issue in the argoproj repo.

Reply:
thanks @shuangkun ! I actually just spoke to @terrytangyuan, and he suggests we drop the requirement to be a member for this altogether. Maybe that makes the most sense since otherwise we run into this kind of problem. Are you good with that @agilgur5 ?

Reply:
I have made this change just now, FYI

Reply:
I think we should at minimum require an open membership request, as that requires a non-trivial level of effort to meet the requirements for (compared to just adding yourself to this list without any contributions). We can call those participants as "pending members"